wasm.md

```diff
@@ -330,6 +354,32 @@ Note that we do not need to call `#chop` on the results here.
     rule <k> ITYPE . rem_s I1 I2 => undefined ... </k> requires I2 ==Int 0
 ```
 
+### Floating Point Arithmetic
+
+**TODO**: Unimplemented
+
+```k
+    syntax FUnOp ::= "neg" | "sqrt" | "floor" | "ceil" | "trunc" | "nearest" | "load" [klabel(floatLoad)]
+ // -----------------------------------------------------------------------------------------------------
+
+    syntax FBinOp ::= "add"   [klabel(floatAdd)]
+                    | "sub"   [klabel(floatSub)]
+                    | "mul"   [klabel(floatMul)]
+                    | "div"   [klabel(floatDiv)]
+                    | "min"   [klabel(floatMin)]
+                    | "max"   [klabel(floatMax)]
+                    | "store" [klabel(floatStore)]
+ // ----------------------------------------------
```

Comment:
What's the purpose of adding klabels here? We don't have them for integer operations. Just curious, as I'm still not all that comfortable with klabels.

Reply:
KLabels are the actual nodes that get placed in the AST after parsing. The issue here is that we need to avoid it using the same klabels for `Float` productions as it does for `Int` productions, or sort-checking will have a hard time.